### DISCUSSION TITLE
Não encontrei SDKS .NET no POWERSHELL

### DISCUSSION THREAD

**Discussion by ThalesHilario** [balta-io/2801#90] [Perguntas e Respostas] [answered]
Ao realizar o teste não aparece nada, nenhuma versão, como se não tivesse sido instalado.

_**PS C:\Users\Thales Hilario> dotnet --list-sdks
PS C:\Users\Thales Hilario>**_

_**It was not possible to find any installed .NET Core SDKs
  Did you mean to run .NET Core SDK commands? Install a .NET Core SDK from:
      https://aka.ms/dotnet-download**_

Alguém sabe como solucionar?

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @ThalesHilario, isto ocorre quando você instala uma SDK errada... por exemplo... instalar a versão ARM ao invés da x64.
Outro problema comum que ocasiona isto é mudar o caminho onde a SDK é instala... veja se foi um dos seus casos.

**Reply by ThalesHilario** [reply]
Valeu, amigo.
Tentei verificar o local, reinstalei, instalei outras versões, mas não funcionou do mesmo jeito.
Foi instalado na pasta C:

**Reply by ThalesHilario** [reply]
_PS C:\Users\Thales Hilario> dotnet

Usage: dotnet [options]
Usage: dotnet [path-to-application]

Options:
  -h|--help         Display help.
  --info            Display .NET Core information.
  --list-sdks       Display the installed SDKs.
  --list-runtimes   Display the installed runtimes.

path-to-application:
  The path to an application .dll file to execute.
PS C:\Users\Thales Hilario> dotnet --list-sdks
PS C:\Users\Thales Hilario>_

**Reply by andrebaltieri** [reply]
Se você instalou em um diretório diferente, precisa atualizar as variáveis globais do Windows (Path) apontando para este novo destino.

https://docs.microsoft.com/pt-br/dotnet/core/install/how-to-detect-installed-versions?pivots=os-windows

**Reply by ThalesHilario** [reply]
Muito obrigado, consegui fazer a instalação e funcionou corretamente.

**Comment by leonardopaeslp** [comment]
Infelizmente ao instalar a versão 6 no Windows 11 estou tendo o mesmo problema.

**Reply by leonardopaeslp** [reply]
Ele estava instalando no C:\Program Files, mas só pegava as que estavam instaladas na C:\Program Files (x86). Por algum motivo, quando copiei o conteúdo da pasta de uma para outra, o Win passou a reconhecer as duas. Então removi a do C:\Program Files (x86) e deixei somente na C:\Program Files que aparentemente é o padrão.

**Reply by ThalesHilario** [reply]
Eu desinstalei através do instalador original, apaguei as pastas em ambos 'Arquivos de Programas' e reinstalei, aí funcionou.
Não sei se foi um acaso, mas deu certo.
